File: programacao/python/respostas/selecao/19.py
mes = int(input("Digite o numero do mes: "))

if mes == 2:
    print("28 dias")
elif mes in (4, 6, 9, 11):
    print("30 dias")
elif mes in (1, 3, 5, 7, 8, 10, 12):
    print("31 dias")
else:
    print("Mes invalido")

File: programacao/c/respostas/selecao/19.c
#include <stdio.h>

int main(void) {
    int mes;

    printf("Digite o numero do mes: ");
    scanf("%d", &mes);

    if (mes == 2) {
        printf("28 dias\n");
    } else if (mes == 4 || mes == 6 || mes == 9 || mes == 11) {
        printf("30 dias\n");
    } else if (mes == 1 || mes == 3 || mes == 5 || mes == 7 || mes == 8 || mes == 10 || mes == 12) {
        printf("31 dias\n");
    } else {
        printf("Mes invalido\n");
    }

    return 0;
}